src/geometries/TubeGeometry.js

```diff
@@ -219,5 +219,14 @@ function TubeBufferGeometry( path, tubularSegments, radius, radialSegments, clos
 TubeBufferGeometry.prototype = Object.create( BufferGeometry.prototype );
 TubeBufferGeometry.prototype.constructor = TubeBufferGeometry;
 
+TubeBufferGeometry.prototype.toJSON = function () {
+
+	var data = BufferGeometry.prototype.toJSON.call( this );
+
+	data.path = this.parameters.path.toJSON();
+
+	return data;
+
+};
```

Comment:
It's better to remove the change to `TubeGeometry` and `ObjectLoader` since you already created a PR for them here: #15789

It's sufficient to note that this PR should merged after #15789

Reply:
Ok. Will do.